salon_creator/lib/screens/lesson_creation_screen.dart

```diff
@@ -21,32 +21,44 @@ class LessonCreationScreen extends StatefulWidget {
 }
 
 class _LessonCreationScreenState extends State<LessonCreationScreen> {
-  bool _publish = false;
-  bool isFormFilled = false;
-  bool operationInProgress = false;
-  List<File> resources = [];
-  List<String> resourcesList = [];
-  TextEditingController categoriesController = TextEditingController();
-  TextEditingController descriptionController = TextEditingController();
-  TextEditingController lessonNameController = TextEditingController();
+  bool _isPublish = false;
```

Comment:
```suggestion
  bool _isPublished = false;
```

Sorry for being super picky